lib/fizzy/execute.cpp

```diff
@@ -293,6 +299,59 @@ inline uint64_t popcnt64(uint64_t value) noexcept
     return static_cast<uint64_t>(__builtin_popcountll(value));
 }
 
+template <typename T>
+inline T fabs(T input) noexcept
```

Comment:
Usually called `value` in other functions
```suggestion
inline T fabs(T value) noexcept
```